README.md

```diff
@@ -1,4 +1,4 @@
-<p align="center"><img src="screenshots/header.png"></p>
+![image](./screenshots/header.png)
```

Comment:
try add `{: .center}` to align the image 

`![image](./screenshots/header.png){: .center}`